### ISSUE TITLE
pip installing something different 

### ISSUE THREAD

**Issue by yashSingh97** [niranjanneeru/cute-text#7]
Hi, so I was looking around and downloaded the module via pip as stated in docs and it was giving errors when i was running the commands in docs.
1. The command to execute tiny/bubble/black/bubble is not CuteTextConverter().convert for the module pip installed, rather it works if we assign .get_converter to assign tiny/bubble to a variable and use .convert in that variable. 
```
tiny = CuteTextConverter()._get_converter(TINY_TEXT)
print(tiny.convert("Hello"))
```
2. Missing import in __init__.py in `C:\Users\<>\AppData\Local\Programs\Python\Python311\Lib\site-packages\cutetext`. Initially running the module, I found import error so I went in and found tiny/black/bubble imports were missing from in file.

**Comment by niranjanneeru** [comment]
Hi, Thanks for notifying this.

Solved this in v0.0.2 release

try

```
pip install cutetext==0.0.2
```

```python
from cutetext import CuteTextConverter
from cutetext import TINY_TEXT, BUBBLE_TEXT, BLACK_BUBBLE_TEXT
print(CuteTextConverter().convert("Hello World!", TINY_TEXT))
print(CuteTextConverter().convert("Hello World!", BUBBLE_TEXT))
print(CuteTextConverter().convert("Hello World!", BLACK_BUBBLE_TEXT))
```

**Comment by yashSingh97** [comment]
Works @niranjanneeru 
I believe this issue has been resolved, so would you mind closing the issue?
